### PR TITLE
chop footer anchors into blocks (bug 1135189)

### PR DESCRIPTION
**Pull request by spasovski**
Tested that it looks the same as current on tablet + desktop.

### CODE REVIEW

File: src/templates/footer.html [hunk 1]
```diff
@@ -2,8 +2,8 @@
   {% include "newsletter.html" %}
 </div>
 
-<div id="footer">
-  <ul class="support-navigation">
+<div id="footer" class="global-footer">
```
Comment (ngokevin) [inline src/templates/footer.html]: could probably just get away with footer

File: src/media/css/footer.styl [hunk 2]
```diff
@@ -42,54 +36,53 @@ $support-nav-height = 55px;
...
+    .footer-extras {
+        float: right;
+        font-size: 14px;
+        line-height: 1.3;
+        margin: 0;
+        padding-left: 1em;
+        width: 20%;
+    }
+    .footer-support-navigation {
+        align-items: baseline;
```
Comment (ngokevin) [inline src/media/css/footer.styl]: need a mixin?

**Comment by ngokevin** [comment]
r+wc

File: src/media/css/footer.styl (hunk 2 above)
Comment (cvan) [inline src/media/css/footer.styl]: Make sure this works in old Firefox (< 24 I believe) and old WebKit (Safari on iOS for sure).

File: src/templates/footer.html [hunk 3]
```diff
@@ -2,8 +2,8 @@
   {% include "newsletter.html" %}
 </div>
 
-<div id="footer">
-  <ul class="support-navigation">
+<div id="footer" class="footer">
+  <ul class="footer-support-navigation">
```
Comment (cvan) [inline src/templates/footer.html]: nice, I like that you're adding classes

File: src/media/css/footer.styl [hunk 4]
```diff
@@ -42,54 +36,53 @@ $support-nav-height = 55px;
...
 }
 
 @media $wide-desktop {
-    #footer {
-        .pad,
-        .support-navigation {
-            width: $desktop-content;
-        }
+    .footer-content,
+    .footer-support-navigation {
```
Comment (cvan) [inline src/media/css/footer.styl]: a class would suffice

File: src/media/css/footer.styl (hunk 2 above)
Comment (ngokevin) [inline src/media/css/footer.styl]: Yeah, I would've used a mixin here to make sure old browsers are addressed @spasovski

**Pull request merged**